src/utils/url.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Checks if a passed in url can be opened in the application.
+ *
+ * @param {string} meetingUrl
+ * @param {Array<string>} validHosts
+ * @return {boolean}
+ */
+export function isValidMeetingUrl(meetingUrl, validHosts) {
+    try {
+        const { domain, host, roomName } = parseMeetingUrl(meetingUrl);
+
+        return domain && roomName && validHosts.includes(host);
+    } catch (e) {
+        return false;
+    }
+}
+
+/**
+ * Parses a meeting url so its parts can be accessed through meeting-related
```

Comment:
can you give an example of a url arg and then how it is split into domain/host/roomname